compose/runtime/runtime/src/commonMain/kotlin/androidx/compose/runtime/Expect.kt

```diff
@@ -105,4 +106,11 @@ internal expect class SnapshotContextElementImpl(
     snapshot: Snapshot
 ) : SnapshotContextElement
 
+/**
+ * Represents a platform-optimized cancellation exception.
+ * This allows us to configure exceptions separately on JVM and other platforms.
+ */
+expect abstract class PlatformOptimizedCancellationException(message: String? = null) :
```

Comment:
> made it internal and copy to all modules where it is used.

I would make it internal because of the high probability of changing

Reply:
Done, made it internal